packages/core/src/tree-view/tree-item.element.ts

```diff
@@ -75,6 +75,14 @@ export class CdsTreeItem extends LitElement {
   @property({ type: Boolean, reflect: true })
   expanded = false;
 
+  /**
+   * @description
+   *
+   * Expandable is used (see line 124) to automatically show the cds-button-expand element when a tree-item detects that
```

Comment:
These are the notes for future dev selves to describe the manual use case if a bug gets reported when a consumer puts something like the `*ngIf` structural directive on one of these elements with children.